scene/2d/audio_stream_player_2d.cpp

```diff
@@ -385,7 +283,11 @@ void AudioStreamPlayer2D::_set_playing(bool p_enable) {
 }
 
 bool AudioStreamPlayer2D::_is_active() const {
-	return active.is_set();
+	if (stream_playback.is_valid()) {
+		// TODO make sure this doesn't change any behavior w.r.t. pauses. Is a paused stream active?
```

Comment:
Just putting a comment so that we remember to evaluate this (either in this PR or as a follow-up, if so might warrant opening an issue so we don't forget).